<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_part_3_1</class>
 <widget class="QDialog" name="fct_part_3_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscription dans une épreuve</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../342px-Olympic_rings_without_rims.svg.png</normaloff>../../../../../../../342px-Olympic_rings_without_rims.svg.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Num Epreuve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_1_numEp"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>||</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Num Equipe/Num Sportif</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_part_3_1_numSpEq"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Les listes des equipes et des sportifs sont automatiquement filtrées: Seulement les équipes/sportifs eligibles sont disponibles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Valider les inscriptions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_part_3_1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_part_3_1_numEp</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_part_3_1</receiver>
   <slot>initialiser_participants()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part_3_1</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initialiserInscrits()</slot>
  <slot>insertResult()</slot>
  <slot>initialiser_participants()</slot>
  <slot>refreshResult()</slot>
 </slots>
</ui>
